scripts/bb-dependencies.sh

```diff
@@ -74,7 +74,7 @@ CentOS*)
     sudo -E yum -y install kernel-devel \
         zlib-devel libuuid-devel libblkid-devel libselinux-devel \
         xfsprogs-devel libattr-devel libacl-devel libudev-devel \
-        device-mapper-devel openssl-devel libffi-devel pam-devel \
+        device-mapper-devel openssl-devel libffi-devel pam-devel pamtester \
```

Comment:
For CentOS `pamtester` appears to be in the epel repository so you'll need to move this down to the "Testing support libraries and tools" section which enables the epel repository.    It would be nice to move this to the utilities section for all the platforms